src/js/views/patients/patient/sidebar/sidebar_views.js

```diff
@@ -61,21 +61,35 @@ const SidebarView = View.extend({
   },
   onClickMenu() {
     const canEdit = this.model.canEdit();
+    const patientStatus = this.model.getStatus();
+    const canManagePatients = this.getOption('canManagePatients');
 
     const menuOptions = new Backbone.Collection([
       {
+        text: canEdit ? i18n.menuOptions.edit : i18n.menuOptions.view,
         onSelect: bind(this.triggerMethod, this, canEdit ? 'click:patientEdit' : 'click:patientView'),
       },
     ]);
 
+    if (canEdit && canManagePatients) {
+      menuOptions.push({
+        text: patientStatus !== 'active' ? i18n.menuOptions.activate : i18n.menuOptions.inactivate,
+        onSelect: bind(this.triggerMethod, this, 'click:activeStatus'),
+      });
+
+      if (patientStatus !== 'archived') {
+        menuOptions.push({
+          text: i18n.menuOptions.archive,
+          onSelect: bind(this.triggerMethod, this, 'click:archivedStatus'),
+        });
```

Comment:
Archiving a patient seems like a big deal. And something that can't be easily reversed in most cases.

Should we add a `Are you sure?` confirmation modal? Could also help users avoid setting `status = archived` on accident.

Reply:
Definitely could use a modal here

Reply:
Confirmation modal has been added:


https://github.com/RoundingWell/care-ops-frontend/assets/35355575/2a2b04b7-896e-41c7-b246-09fbb42c2c81